Exercise 1 BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,214 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "fileName",
+        type = "BString",
+        defaultValue = "BString.DEFAULT"
+)
+@NiagaraProperty(
+        name = "path",
+        type = "BString",
+        defaultValue = "BString.DEFAULT"
+)
+@NiagaraProperty(
+        name = "size",
+        type = "BInteger",
+        defaultValue = "BInteger.DEFAULT"
+)
+@NiagaraProperty(
+        name = "createdDate",
+        type = "BAbsTime",
+        defaultValue = "BAbsTime.make()"
+)
+@NiagaraProperty(
+        name = "modifiedDate",
+        type = "BAbsTime",
+        defaultValue = "BAbsTime.NULL"

```

Comment:
I would pick a consistent default `BAbsTime` value and use that both here and on line 27. Using `BAbsTime.DEFAULT` is a good choice.